### ISSUE TITLE
Admission controller can't restart after crashing due to volcano webhook failure policy: fail

### ISSUE THREAD

**Issue by djwhatle** [volcano-sh/volcano#2244]
**What happened**:

The `volcano-admission` Pod crashed and tried to start back up, but was denied admission because there is a self-referential dependency for this Pod to be admitted.

**What you expected to happen**:

volcano-admission Pod can crash and restart

**How to reproduce it (as minimally and precisely as possible)**:

```
# First scale down the volcano-admission Pod to simulate a Pod crash or OOM event
kubectl scale deployment volcano-admission -n volcano-system --replicas=0

# Now wait for the volcano-admission pod to stop
kubectl scale deployment volcano-admission -n volcano-system --replicas=1

# Notice the volcano-admission pod won't come back into a running state
kubectl get pod -n volcano-system                                                                                                                                                                                                        

NAME                                   READY   STATUS      RESTARTS   AGE
volcano-admission-init-rgdn2           0/1     Completed   0          54m
volcano-controllers-7c6b5c4f4b-cxvg5   1/1     Running     0          54m
volcano-scheduler-7f9d9984fd-zz52w     1/1     Running     0          54m
```

**Anything else we need to know?**:

**Environment**:
- Volcano Version: v1.5.1
- Kubernetes version (use `kubectl version`): 1.21
- Cloud provider or hardware configuration: GKE
- Install tools: In a kubeflow env


I have a patch for this: simply set the retrypolicy for webhooks to ignore instead of fail when the webhook is not available. The admission controller should be multi-replica or else the failurePolicy on the webhook should not be set to fail.

**Comment by djwhatle** [comment]
Fix implemented in #2245

**Comment by djwhatle** [comment]
Implementing an alternate fix in an upcoming PR. It sounds like there are some potentially harmful side-effects (panic in volcano controller) if webhook `failurePolicy: ignore` is set. 

Arguably the controllers should be resilient to webhook failure without panic but for practical purposes, will plan to open a PR to set the webhooks to ignore all Pod admissions in volcano-system and kube-system namespaces.
